release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -407,6 +407,42 @@ For more information, see xref:../support/troubleshooting/troubleshooting-networ
 [id="ocp-4-12-osdk"]
 === Operator development
 
+[id="ocp-4-12-osdk-security-context-config"]
+==== Configuring the security context of a catalog pod
+
+You can configure the security context of a catalog pod by using the `--security-context-config` flag on the `run bundle` and `bundle-upgrade` subcommands. The flag enables seccomp profiles to comply with pod security admission. The flag accepts the values of `restricted` and `legacy`. If you do not specify a value, the seccomp profile defaults to `restricted`. If your catalog pod cannot run with restricted permissions, set the flag to `legacy`, as shown in the following example:
+
+[source,terminal]
+----
+$ operator-sdk run bundle \
+  --security-context-config=legacy
+----
+
+[id="ocp-4-12-osdk-bundle-validate-removed-kube-1-25-apis"]
+==== Validating bundle manifests for APIs removed in Kubernetes 1.25
+
+You can now check bundle manifests for APIs removed from Kubernetes 1.25 by using the Operator Framework suite of tests with the `bundle validate` subcommand.
+
+For example:
+
+[source,terminal]
+----
+$ operator-sdk bundle validate .<bundle_dir_or_image> \
+  --select-optional suite=operatorframework \
+  --optional-values=k8s-version=1.25
+----
+
+If your Operator requests permission to use any of the APIs removed from Kubernetes 1.25 the command displays a warning message.
+
+[NOTE]
+====
+The `operator-sdk bundle validate` command cannot determine which version of the APIs your Operator requests permission to use. As a result, your Operator might trigger warnings even if you use newer versions of the API.
```

Comment:
I think it may be worth also mentioning that there is a known bug in v1.26.0 that ignores the group and will issue a warning if the resource name matches one that was deprecated. This should be resolved in a future upstream patch release.

@michaelryanpeter WDYT?

Reply:
I can add a note to the _Known issues_ section of the 4.12 release notes. Most of the time, unless it is a major problem, we don't mention bugs or known issues in the docs. It is a big maintance burden once they fixed. I will sync with you and we can put together a description for the release ntoes.

Reply:
I created a ticket to track this work: https://issues.redhat.com/browse/OSDOCS-4746